microservices.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
We shouldn't mix multiple features on the same branch. Please revert this file, and submit separate PR from a separate branch for k8s.